manifests/claudie/dynamo/job.yaml

```diff
@@ -3,12 +3,14 @@ kind: Job
 metadata:
   name: create-table-job
   labels:
-    app: dynamo-create-table-job
+    app.kubernetes.io/part-of: dynamo
+    app.kubernetes.io/name: dynamo/dynamodb
```

Comment:
this is inconsistent and ugly, please go with either `dynamo` or `dynamodb` for all resources under folder `dynamo` in a consistent manner